images/rockylinux.yaml

```diff
@@ -35,6 +35,41 @@ source:
     JmeWf4VZyebm+gc50szsg6Ut2yT8hw==
     =AiP8
     -----END PGP PUBLIC KEY BLOCK-----
+    
+    # RPM-GPG-KEY-rockylinux-9
+    - |-
```

Comment:
I believe this indentation is wrong, and needs to be aligned correctly (needs to be moved to the left).

Reply:
I've moved it see https://github.com/lxc/lxc-ci/pull/576/commits/a6ac9a6482b3dd08813db7c72a5c8a1928e0a9af